lib/projects/localDev/LocalDevManagerV2.ts

```diff
@@ -1,557 +0,0 @@
-import path from 'path';
```

Comment:
Broken up into `LocalDevProcess`, `LocalDevLogger`, `LocalDevWatcher`, and `AppDevModeInterface`